sonyapilib/device.py

```diff
@@ -272,6 +281,32 @@ def _parse_ircc(self):
             self.ircc_url, method=HttpMethod.GET, raise_errors=True)
 
         upnp_device = f"{URN_UPNP_DEVICE}device"
+
+        self.friendly_name = self._find_device_info(response.text, "friendlyName",
```

Comment:
wouldn't it be easier to initalize the field right away instead of splitting decl and init?

Reply:
We need `response.text` from ircc request, so I guess we can't move it to `__init__` ?